src/org/robockets/steamworks/OI.java

```diff
@@ -159,8 +159,8 @@ private void bindButtons() {
         aButton.whenPressed(new IntakeToPos(IntakeFlap.IntakeState.FUEL));
         aButton.whenReleased(new KillLights());
 
-        xButton.whileHeld(new ShootWithPID(50));
-        bButton.whileHeld(new ShootWithPID(55));
+        xButton.whileHeld(new ShootWithPID(63));
+        bButton.whileHeld(new ShootWithPID(80));
```

Comment:
No, but making it higher than it can go seems to make it spin up faster. It works, I don't want to touch it...